docs/source/contributing/roadmap.md

```diff
@@ -0,0 +1,46 @@
+# The repo2docker roadmap
```

Comment:
could we also add a sym-link to this file in the root of the repository? That way we have both the "github developer" types and the documentation-browsing types covered. WDYT?

Reply:
The roadmap is linked from the CONTRIBUTING.md, should we also feature it in the README?

Reply:
Where in the docs should we directly link to it? From the front page at the top where it says "contribute", "open an issue"?

Reply:
Perhaps at this point in the docs: https://repo2docker.readthedocs.io/en/latest/contributing/contributing.html